src/vstart.sh

```diff
@@ -1658,6 +1658,12 @@ fi
 
 if [ "$ceph_osd" == "crimson-osd" ]; then
     $CEPH_BIN/ceph -c $conf_fn config set osd crimson_seastar_smp $crimson_smp
+    if [ "$(expr $(nproc) - 1)" -gt "$(($CEPH_NUM_OSD * crimson_smp))" ]; then
+      echo "crimson_alien_thread_cpu_cores:" $(($CEPH_NUM_OSD * crimson_smp))-"$(expr $(nproc) - 1)"
+      $CEPH_BIN/ceph -c $conf_fn config set osd crimson_alien_thread_cpu_cores $(($CEPH_NUM_OSD * crimson_smp))-"$(expr $(nproc) - 1)"
+    else
+      echo "No alien thread cpu core isolation"
+    fi
```

Comment:
can we assign osd0 0-(smp-1), osd1 smp-(2*smp-1) and etc.?

Reply:
We already assign this in vstart:
```
    for osd in `seq $start $end`
    do
	local extra_seastar_args
	if [ "$ceph_osd" == "crimson-osd" ]; then
        bottom_cpu=$(( osd * crimson_smp ))
        top_cpu=$(( bottom_cpu + crimson_smp - 1 ))
	    # set a single CPU nodes for each osd
	    extra_seastar_args="--cpuset $bottom_cpu-$top_cpu"
	fi
```